src/AlgoliaSearch.js

```diff
@@ -118,54 +119,56 @@ AlgoliaSearch.prototype = {
    *  error: null or Error('message')
    *  content: the server answer that contains the task ID
    */
-  deleteIndex: function(indexName, callback) {
-    return this._jsonRequest({ method: 'DELETE',
-              url: '/1/indexes/' + encodeURIComponent(indexName),
-              hostType: 'write',
-              callback: callback });
+  deleteIndex: function deleteIndex(indexName, callback) {
+    return this._jsonRequest({method: 'DELETE',
+      url: '/1/indexes/' + encodeURIComponent(indexName),
+      hostType: 'write',
+    callback: callback});
   },
   /**
    * Move an existing index.
    * @param srcIndexName the name of index to copy.
-   * @param dstIndexName the new index name that will contains a copy of srcIndexName (destination will be overriten if it already exist).
+   * @param dstIndexName the new index name that will contains a copy of
+   * srcIndexName (destination will be overriten if it already exist).
    * @param callback the result callback called with two arguments
    *  error: null or Error('message')
    *  content: the server answer that contains the task ID
    */
-  moveIndex: function(srcIndexName, dstIndexName, callback) {
+  moveIndex: function moveIndex(srcIndexName, dstIndexName, callback) {
     var postObj = {operation: 'move', destination: dstIndexName};
-    return this._jsonRequest({ method: 'POST',
-              url: '/1/indexes/' + encodeURIComponent(srcIndexName) + '/operation',
-              body: postObj,
-              hostType: 'write',
-              callback: callback });
-
+    return this._jsonRequest({method: 'POST',
+      url: '/1/indexes/' + encodeURIComponent(srcIndexName) + '/operation',
+      body: postObj,
+      hostType: 'write',
+    callback: callback});
   },
   /**
    * Copy an existing index.
    * @param srcIndexName the name of index to copy.
-   * @param dstIndexName the new index name that will contains a copy of srcIndexName (destination will be overriten if it already exist).
+   * @param dstIndexName the new index name that will contains a copy
+   * of srcIndexName (destination will be overriten if it already exist).
    * @param callback the result callback called with two arguments
    *  error: null or Error('message')
    *  content: the server answer that contains the task ID
    */
-  copyIndex: function(srcIndexName, dstIndexName, callback) {
+  copyIndex: function copyIndex(srcIndexName, dstIndexName, callback) {
     var postObj = {operation: 'copy', destination: dstIndexName};
-    return this._jsonRequest({ method: 'POST',
-              url: '/1/indexes/' + encodeURIComponent(srcIndexName) + '/operation',
-              body: postObj,
-              hostType: 'write',
-              callback: callback });
+    return this._jsonRequest({method: 'POST',
+      url: '/1/indexes/' + encodeURIComponent(srcIndexName) + '/operation',
+      body: postObj,
+      hostType: 'write',
+    callback: callback});
```

Comment:
Oh yes you are right this is a mistake, strange it was rewritten this way, will see what I can do